Assets/Scripts/TrainingAgent.cs

```diff
@@ -84,6 +84,7 @@ public override void CollectObservations(VectorSensor sensor)
         sensor.AddObservation(localVel);
         Vector3 localPos = transform.position;
         sensor.AddObservation(localPos);
+        Debug.Log("Health:" + health + "\nVelocity: " + localVel + "\nPosition:" + localPos);
```

Comment:
Debug.Log() simply prints these in the unity console. The researcher will not be able to see them in the .exe, meaning it will be hidden as the unity console is strictly for development and is infused in the editor. 

I can't remember if you mentioned you could obtain the player logs within the folder, which records the debug logs. @kozzy97